lib/Screens/login_page.dart

```diff
@@ -0,0 +1,133 @@
+
+import 'package:flutter/material.dart';
+import '../api//auth_user.dart';
+import '../../Screens/profile_screen.dart';
+
+
+class LoginPage extends StatefulWidget {
+  final Auth api=Auth();
+  LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+
+
+final usernameController= TextEditingController();
+final passwordController = TextEditingController();
+
+
+  @override
+  Widget build(BuildContext context) {
+
+    return MaterialApp(
+        home: Scaffold(
+          backgroundColor: Colors.white70,
+          appBar: AppBar(title: const Text("Login Page"),
+          backgroundColor: const Color.fromRGBO(170, 126, 189, 100),),
+          body: Center(
+            child: Stack(
+              children:<Widget> [
+
+          Center(
+          child :Padding(
+              padding: const EdgeInsets.all(20),
+              child: Container(
+                width: MediaQuery.of(context).size.height * 0.4,
+                height: MediaQuery.of(context).size.height * 0.7,
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                  borderRadius: BorderRadius.circular(20),
+                ),
+                child: Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 75),
+                child: Center(
+                  child: Column(
+                    children: [
+                      const SizedBox(
+                        height: 15,
+                      ),
+                      const Image(
+                          image: AssetImage('images/3898671.png')),
+                      const Text(
+                        "Login",
+                        style: TextStyle(
+                            color: Color.fromRGBO(170, 126, 189, 100),
+                            fontWeight: FontWeight.bold,
+                            fontSize: 24),
+                      ),
+                      const SizedBox(
+                        height: 45,
+                      ),
+                      Padding(padding: const EdgeInsets.symmetric(horizontal: 50,vertical: 20),
+                      child:TextField(
```

Comment:
It is good practice to have form validations to prevent user from submitting invalid requests. You can use the Form widget along with TextFormField to achieve this